src/Product/FinancialDocument/FinancialDocumentV1LineItem.php

```diff
@@ -100,16 +114,16 @@ private function printableValues(): array
      */
     public function toTableLine(): string
     {
-        $printable = $this->printableValues();
+        $printable = $this->tablePrintableValues();
         $outStr = "| ";
-        $outStr .= str_pad($printable["description"], 36) . " | ";
-        $outStr .= str_pad($printable["productCode"], 12) . " | ";
-        $outStr .= str_pad($printable["quantity"], 8) . " | ";
-        $outStr .= str_pad($printable["taxAmount"], 10) . " | ";
-        $outStr .= str_pad($printable["taxRate"], 12) . " | ";
-        $outStr .= str_pad($printable["totalAmount"], 12) . " | ";
-        $outStr .= str_pad($printable["unitMeasure"], 15) . " | ";
-        $outStr .= str_pad($printable["unitPrice"], 10) . " | ";
+        $outStr .= mb_str_pad($printable["description"], 36, ' ', STR_PAD_RIGHT, "UTF-8") . " | ";
+        $outStr .= mb_str_pad($printable["productCode"], 12, ' ', STR_PAD_RIGHT, "UTF-8") . " | ";
+        $outStr .= mb_str_pad($printable["quantity"], 8, ' ', STR_PAD_RIGHT, "UTF-8") . " | ";
+        $outStr .= mb_str_pad($printable["taxAmount"], 10, ' ', STR_PAD_RIGHT, "UTF-8") . " | ";
+        $outStr .= mb_str_pad($printable["taxRate"], 12, ' ', STR_PAD_RIGHT, "UTF-8") . " | ";
+        $outStr .= mb_str_pad($printable["totalAmount"], 12, ' ', STR_PAD_RIGHT, "UTF-8") . " | ";
+        $outStr .= mb_str_pad($printable["unitMeasure"], 15, ' ', STR_PAD_RIGHT, "UTF-8") . " | ";
+        $outStr .= mb_str_pad($printable["unitPrice"], 10, ' ', STR_PAD_RIGHT, "UTF-8") . " | ";
```

Comment:
Lots of duplicate code, I would probably create a static method in `SummaryHelper` (called for each field), but up to you.